specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-03-01/managedClusters.json

```diff
@@ -1997,7 +1997,7 @@
           "type": "object",
           "properties": {
             "balance-similar-node-groups": {
-              "type": "boolean"
```

Comment:
>boolean [](start = 23, length = 7)

I discussed this over IM and while this appears to be a breaking change, this property never worked right in the service. As a result, I'm considering this more of a swagger bug fix.

From past API reviews, I recall this was basically a property bag of string properties, so coercing the bool into a string is fine here.